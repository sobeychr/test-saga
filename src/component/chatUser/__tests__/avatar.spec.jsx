import React from 'react';
import { shallow } from 'enzyme';
import { userMock } from 'Util/mock';
import toJson from 'enzyme-to-json';

import Avatar from './../avatar';

describe('src/component/charUser/avatar', () => {
    const title = 'test-title';

    it('should render properly', () => {
        const wrapper = shallow(<Avatar user={userMock} title={title} />);
        const classes = wrapper.prop('className');

        expect(wrapper).toHaveLength(1);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
