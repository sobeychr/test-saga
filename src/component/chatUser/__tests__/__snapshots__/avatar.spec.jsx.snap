// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/component/charUser/avatar should render properly 1`] = `
<div
  className="avatar avatar_4"
  title="test-title"
/>
`;
